### ISSUE TITLE
Problem with entity_load_multiple_by_properties()

### ISSUE THREAD

**Issue by bjaxelsen** [agentrickard/domain#70]
I got this error message: 

> Fatal error: Call to undefined function Drupal\domain\entity_load_multiple_by_properties() in /dana/data/betadrupal8/docs/modules/domain/domain/src/DomainLoader.php on line 94

I am  not sure why it cannot use the function, but I made it work by change the code:

```
diff --git a/domain/src/DomainLoader.php b/domain/src/DomainLoader.php
index f073fe3..54dba2d 100644
--- a/domain/src/DomainLoader.php
+++ b/domain/src/DomainLoader.php
@@ -53,7 +53,9 @@ class DomainLoader implements DomainLoaderInterface {
    * {@inheritdoc}
    */
   public function loadDefaultId() {
-    $result = entity_load_multiple_by_properties('domain', array('is_default' =
+    $result = \Drupal::entityManager()
+      ->getStorage('domain')
+      ->loadByProperties(array('is_default' => TRUE));
     if (!empty($result)) {
       return key($result);
     }
@@ -64,7 +66,9 @@ class DomainLoader implements DomainLoaderInterface {
    * {@inheritdoc}
    */
   public function loadDefaultDomain() {
-    $result = entity_load_multiple_by_properties('domain', array('is_default' =
+    $result = \Drupal::entityManager()
+      ->getStorage('domain')
+      ->loadByProperties(array('is_default' => TRUE));
     if (!empty($result)) {
       return current($result);
     }
@@ -91,7 +95,9 @@ class DomainLoader implements DomainLoaderInterface {
    * {@inheritdoc}
    */
   public function loadByHostname($hostname) {
-    $entities = entity_load_multiple_by_properties('domain', array('hostname' =
+    $entities = \Drupal::entityManager()
+      ->getStorage('domain')
+      ->loadByProperties(array('hostname' => $hostname));
     if (empty($entities)) {
       return FALSE;
     }
```

**Comment by agentrickard** [comment]
The function has been deprecated. See https://api.drupal.org/api/drupal/core!includes!entity.inc/function/entity_load_multiple_by_properties/8

**Comment by agentrickard** [comment]
Fixed by #71
